### ISSUE TITLE
issue with continuing installation of wirepod on a raspberry pi.

### ISSUE THREAD

**Issue by SgtKiLLx** [kercre123/wire-pod#215]
upon attempting to install, it appears its not downloading GO. ive installed it manually, and still getting the same issue.

![Screenshot 2023-10-29 110415](https://github.com/kercre123/wire-pod/assets/70082342/44f99f4e-1623-41b3-913e-322b823c937a)
![Screenshot 2023-10-29 112147](https://github.com/kercre123/wire-pod/assets/70082342/cbe1cb39-2bfc-4631-b11f-9bb6db1186b4)

**Comment by F4310** [comment]
What is the operating system?

**Comment by F4310** [comment]
Try using Armbian 23.02.2 Bullseye

**Comment by SgtKiLLx** [comment]
> What is the operating system version?

Latest Debian

**Comment by kercre123** [comment]
Make sure you're using a 64-bit Raspberry Pi OS. The default Raspberry Pi OS has a 64-bit kernel, but links with 32-bit, and setup.sh can't detect that at the moment.

**Comment by SgtKiLLx** [comment]
Will try.

**Comment by SgtKiLLx** [comment]
alright. chipper running. as per the following step(s)

[Authenticate a production bot]

It is recommended to clear your bot's user data. This is not required, and you can still authenticate with wire-pod without it (as long as the last server you have authenticated the bot with was the DDL/Anki production stack), but it may cause unexpected behavior.

Place Vector on the charger
Double press his button
Lift his lift up then down
Take Vector off of the charger and twist one of the wheels until the cursor is on "RESET" (or "CLEAR USER DATA")
Lift the lift up then down again
Move the wheel until the cursor is on "CONFIRM"
Lift the lift up then down again

**Refresh** the [vector-wirepod-setup](https://keriganc.com/vector-wirepod-setup) page and follow the instructions  <<<<<-----  I get to here, refreshing does nothing. 

![Screenshot 2023-11-06 202153](https://github.com/kercre123/wire-pod/assets/70082342/17da5259-af41-47cc-bbe5-30e045e4238e)

with V all brand new, with a new ID#, hes jus sittin on his charger with his head up with anki.com/v across his face

**Comment by kercre123** [comment]
Follow the instructions on the page. It will detect that Vector isn't in recovery mode and will authenticate him rather than put a different firmware on him.

**Comment by jowensfog** [comment]
Just wanted to say I ran into this same issue and following the instructions, was able to get it resolved. I have a brand new raspberry pi 4b that came with a pre-installed OS. I ended up downloading the latest 64bit os and started from scratch and that helped.
And then during the setup stage I also got Vector into the state described above. I realized I needed to double-tap Vector's button to get into pair mode. And then after updating the firmware when that Pair With Vector button was shown, to then just double-tap Vector's button again. On the first go around I made the mistake of holding his button down for 15 seconds to pair him again (which I think reset his firmware). That may have been obvious to others, but thought I'd say that here in case anyone else new makes that mistake.
